src/contracts/pods/EigenPod.sol

```diff
@@ -712,6 +743,10 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         address recipient,
         uint64 partialWithdrawalAmountGwei
     ) internal returns (VerifiedWithdrawal memory) {
+        require(
```

Comment:
why do we need this?

Reply:
won't it be enabled during the upgrade and paused via regular pausing method?

Reply:
bump on this @Sidu28 
I had the same questions.